src/pages/billing/Project.js

```diff
@@ -225,7 +225,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const signal = useCancellation()
 
-  const adminCanEdit = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
+  const projectHasMultipleOwners = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
```

Comment:
The previous name for this was misleading and I think resulted in it being used in some ways that it shouldn't have been. The previous name implied that the user had admin privileges on the project. In reality it just checks that the project won't be orphaned if you remove the only owner.